src/features/orbitalfeature.jl

```diff
@@ -0,0 +1,77 @@
+include("abstractfeatures.jl")
+using DataFrames
+using ..ChemistryFeaturization: elements
+using ..ChemistryFeaturization.AbstractType: AbstractCodec, AbstractAtoms
+using ..ChemistryFeaturization.Codec: SimpleCodec
+using ..ChemistryFeaturization.Utils.OrbitalFeatureUtils:
+    valence_shell_config, _indexorbital
+using SparseArrays
+
+struct OrbitalFeatureDescriptor <: AbstractEnvironmentFeatureDescriptor
+    lookup_table::DataFrame
+    encoder_decoder::AbstractCodec
+    OrbitalFeatureDescriptor() =
+        new(atom_data_df, SimpleCodec(default_ofd_encode, default_ofd_decode))
+end
+
```

Comment:
> can you add pretty printing for this type somewhere? Currently it takes forever because it prints out the whole data table :P

Haha I think at one point I had some hacky pretty printing set up that said something like `OrbitalFeatureDescriptor created!`, but had it removed because it wasn't really contributing any useful information. But I guess we can bring it back with something similar getting printed out?

> Related: for memory purposes, can the constructor trim down the table to only have the info it needs to decode rather than all of it?

yes that sounds like a good idea. I was considering it myself, but left it as is in case I realized later on I needed some other value from another column. But that seems unlikely. Will do this.

Reply:
Resolved in aed14ba.

Reply:
as for the pretty printing, I think even if it just says `OrbitalFeatureDescriptor` alone, that's probably fine. Just something so that if it is inspected, or appears in a list with other FD's or whatever, you don't have that whole DataFrame getting spewed to REPL 😆

Reply:
I'm also questioning whether a lookup table even needs to be explicitly stored...we could just extend the built-in one to have every element (the ones I excluded before were pretty arbitrary as things that don't _usually_ appear in compounds, but it doesn't hurt to just have everything), and then we shouldn't need it at all...

Reply:
yes, that makes sense to me too. But I figured that it might be easier to have a `DataFrame` field, as you wouldn't have to filter `atom_data_df` _everytime_ you want to encode or decode.
An alternate idea I could think of is, moving `atom_data_df` out of `ElementFeatureUtils` (as even `OrbitalFeatureDescriptor` uses it now) into its own module/at least file, and have a new `DataFrame` which  is essentially `atom_data_df` filtered to have only the `Symbol` and `Electronic Configuration` (valence shell only but) columns. Then, `OrbitalFeatureDescriptor` could use this directly instead. Does that make sense?

Reply:
yeah it probably would make sense to move it, maybe to a "data" module or something since it's now being used by something other than just EFD. I'm good with that solution!

Reply:
Resolved in a32599d and 5a732f8 respectively.